downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md

```diff
@@ -226,30 +226,30 @@ public function main() {
 
 ### HTTP
 
-The redirect client has been improved to support temporary and permanent redirects, with the original request’s HTTP method. A new `allowAuthHeaders` field was introduced in the redirect configuration to get user consent before repeating sensitive data on subsequent requests.
+The redirect client has been improved to support temporary and permanent redirects with the original request’s HTTP method. A new `allowAuthHeaders` field was introduced in the redirect configuration to get user consent before repeating sensitive data on subsequent requests.
 
 ### Log
 
-A function  log:setModuleLogLevel has been introduced  to allow users to set the module log level through the Log API.
+A `log:setModuleLogLevel` function has been introduced to allow users to set the module log level through the Log API.
 
 ### Config
 
-The `config:getAsArray` and `config:getAsMap` functions now return immutable values. This way the user can simply cast the returned value to the desired list type or mapping type without having to use `cloneWithType()`.
+The `config:getAsArray` and `config:getAsMap` functions now return immutable values. This way, the user can simply cast the returned value to the desired list type or mapping type without having to use `cloneWithType()`.
 
 ### Security
 
-- JWT signature validation with JWK has been improved by introducing a cache. This cache preloads the JWKs and thereby reduces the runtime latency added due to the HTTP call to the JWK endpoint, when validating the signature.
-- An auth:InvocationContext record which can be used as a data holder has been introduced. It can hold auth-related information such as authentication scheme, auth token, and authenticated user's ID, claims, and scopes preserved for a single request-response flow.
+- JWT signature validation with JWK has been improved by introducing a cache. This cache preloads the JWKs, and thereby, reduces the runtime latency added due to the HTTP call to the JWK endpoint when validating the signature.
+- An `auth:InvocationContext` record, which can be used as a data holder has been introduced. It can hold auth-related information such as authentication scheme, auth token, and authenticated user's ID, claims, and scopes preserved for a single request-response flow.
 
 ### WebSocket
 
-The WebSocket client now supports cookies. A new field `cookies` has been introduced in `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` to specify the cookies.
+The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` to specify the cookies.
```

Comment:
If we are adding "the" shouldn't this be

```suggestion
The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` records to specify the cookies.
```

`http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` are names.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1072.